app/subsonic/subsonic.js

```diff
@@ -129,35 +129,23 @@ angular.module('jamstash.subsonic.controller', [
 
     $rootScope.showIndex = false;
     $scope.toggleIndex = function () {
-        if ($rootScope.showIndex) {
-            $rootScope.showIndex = false;
-        } else {
-            $rootScope.showIndex = true;
-            $scope.showPlaylist = false;
-            $scope.showPodcast = false;
-        }
+        $rootScope.showIndex = true;
+        $scope.showPlaylist = false;
+        $scope.showPodcast = false;
         $scope.saveDefaultSection('index');
     };
     $scope.showPlaylist = false;
     $scope.togglePlaylist = function () {
-        if ($scope.showPlaylist) {
-            $scope.showPlaylist = false;
-        } else {
-            $scope.showPlaylist = true;
-            $rootScope.showIndex = false;
-            $scope.showPodcast = false;
-        }
+        $scope.showPlaylist = true;
+        $rootScope.showIndex = false;
+        $scope.showPodcast = false;
```

Comment:
Hey I know it wasn't the case before but what do you think about keeping the var orders ? That way it should be easier to read.

$rootScope.showIndex = false;
$scope.showPlaylist = true;
$scope.showPodcast = false;

Reply:
Done!